src/types/navigation.tsx

```diff
@@ -7,6 +7,7 @@ import { NativeStackScreenProps } from '@react-navigation/native-stack';
 
```

Comment:
Not related to your changes, but I noticed there's an eslint-disable-next-line on line 5 of this file that I'm pretty sure doesn't need to be there, since the package is indeed imported in package.json